StudioCore/MsbEditor/PropertyEditor.cs

```diff
@@ -391,7 +391,7 @@ private bool ParamRefRow(PropertyInfo propinfo, object oldval, ref object newObj
                 refs.Add(att.ParamName);
                 Editor.EditorDecorations.ParamRefText(refs);
                 ImGui.NextColumn();
-                var id = (int)oldval;
+                var id = oldval; //oldval cannot always be casted to int
```

Comment:
this feels like a band-aid to the problem. We should ideally either convert every type, or filter out those that don't match

Reply:
Does it really need to be cast in the first place though? It didn't seem like it did, maybe I'm overlooking something though. I left the stripped line + comment in there in case I was wrong.